robotiq_s_model_articulated_gazebo_plugins/include/grizzly_ur10_gazebo_plugins/RobotiqHandPlugin.h

```diff
@@ -0,0 +1,288 @@
+/*
```

Comment:
That's right. Thanks for the catch.
On Jan 18, 2015 6:15 AM, "G.A. vd. Hoorn" <notifications@github.com> wrote:

> In
> robotiq_s_model_articulated_gazebo_plugins/include/grizzly_ur10_gazebo_plugins/RobotiqHandPlugin.h
> <https://github.com/ros-industrial/robotiq/pull/17#discussion-diff-23131658>
> :
>
> > @@ -0,0 +1,288 @@
> > +/*
>
> Forgot to change directory name (include/grizzly_ur10_gazebo_plugins)?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ros-industrial/robotiq/pull/17/files#r23131658>.
>